.ci/test.sh

```diff
@@ -51,11 +51,23 @@ if [[ $TRAVIS == "true" ]] && [[ $TASK == "check-docs" ]]; then
 fi
 
 if [[ $TASK == "lint" ]]; then
-    conda install -q -y -n $CONDA_ENV pycodestyle pydocstyle
+    conda install -q -y -n $CONDA_ENV \
+        pycodestyle \
+        pydocstyle \
+        r-lintr
     pip install --user cpplint
+    echo "Linting Python code"
     pycodestyle --ignore=E501,W503 --exclude=./compute,./.nuget . || exit -1
     pydocstyle --convention=numpy --add-ignore=D105 --match-dir="^(?!^compute|test|example).*" --match="(?!^test_|setup).*\.py" . || exit -1
-    cpplint --filter=-build/c++11,-build/include_subdir,-build/header_guard,-whitespace/line_length --recursive ./src ./include || exit 0
+    echo "Linting R code"
+    Rscript ${BUILD_DIRECTORY}/.ci/lint_r_code.R ${BUILD_DIRECTORY} || exit -1
+    echo "Linting C++ code"
+    cpplint \
+        --filter=-build/include_subdir,-build/header_guard,-whitespace/line_length \
```

Comment:
Bad rebase: `-build/c++11` is lost.

Reply:
oh no! sorry, good catch. will update it.

Reply:
This has been updated. I removed the `\` so the `cpplint` line is now completely untouched. Mind taking one more look? I can't merge with pending changes. Thanks!